README.md

```diff
@@ -362,6 +362,7 @@ Discover the full list of Nuxt modules on https://modules.nuxtjs.org
 
 ![Nuxt.js v2][nuxt-v2-label] ![Nuxt.js v1][nuxt-v1-label]
 
+- [Nuxt.js — Two super easy changes for starters that can dramatically improve the performance](https://lukeliutingchun.medium.com/nuxt-js-two-super-easy-change-that-can-dramatically-improve-the-performance-c954ee171d02)
```

Comment:
To respect the other link, please move it at the end of the section 😉

Reply:
OK I didn't notice that 😀